src/InterfaxChannel.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace NotificationChannels\Interfax;
+
+use Illuminate\Notifications\Notification;
+use Interfax\Client;
+use NotificationChannels\Interfax\Exceptions\CouldNotSendNotification;
+
+class InterfaxChannel
+{
+    protected $client;
+
+    public function __construct(Client $client)
+    {
+        $this->client = $client;
+    }
+
+    /**
+     * Send the given notification.
+     *
+     * @param mixed $notifiable
+     * @param \Illuminate\Notifications\Notification $notification
+     *
+     * @throws \NotificationChannels\Interfax\Exceptions\CouldNotSendNotification
+     */
+    public function send($notifiable, Notification $notification)
+    {
+        if (! $faxNumber = $notifiable->routeNotificationFor('Interfax')) {
```

Comment:
```suggestion
        if (! $faxNumber = $notifiable->routeNotificationFor('interfax')) {
```

Lowercase route please :)